### DISCUSSION TITLE
Frage zur Verwendung des Battery Icons

### DISCUSSION THREAD

**Discussion by Sorbat** [hoylabs/OpenDTU-OnBattery#860] [General]
Hallo @helgeerbe und alle anderen, 

ich wollte mich mal erkundigen, ob es möglich wäre das Battery Icon verwenden zu können und falls das möglich wäre, in welchem Umfang, da ich es ganz gerne auf meinem erstellten DTUoB Gehäuse platzieren würde. 

Hier mal ein erster Entwurf von dem, was ich mir so vorgestellt hatte. 
<img width="454" alt="Bildschirmfoto 2024-04-09 um 16 53 43" src="https://github.com/helgeerbe/OpenDTU-OnBattery/assets/72571258/d55a638f-362e-455a-bbb0-68ef3fdb4e57">
<img width="464" alt="Bildschirmfoto 2024-04-09 um 17 00 28" src="https://github.com/helgeerbe/OpenDTU-OnBattery/assets/72571258/4a29156c-487b-460f-8a00-ec25c09a2429">
Entweder als negativ oder ggfs. auch zweifarbig. Ein passender Drucker für zweifarbigen Druck ist aktuell noch nicht vorhanden, wäre also dann ein Grund für eine neue Anschaffung :)  Man müsste sich aber generell mal anschauen, wie es gedruckt aussieht, weil es ja doch schon ziemlich klein ist. 
Wie ist da dein/euer Standpunkt zu einer privaten oder ggfs. auch einer kommerziellen Verwendung des Icons? 
Ich hatte zwar gesehen, dass das Projekt unter GPL-2.0 lizensiert ist, allerdings kenne ich mich damit nicht aus und weiß auch nicht, ob sich das eventuell nur auf die Software bezieht etc. pp. Wenn ich mich da durch die Lizenzbedingungen kämpfen muss, verwerfe ich die Idee lieber.
 
Würde mich über eine kurze Rückmeldung freuen. 

Vielen Dank und viele Grüße

**Comment by schlimmchen** [comment]
https://icons.getbootstrap.com/icons/battery-charging/
https://getbootstrap.com/docs/5.2/extend/icons/
https://opensource.org/license/mit

Ich kann nicht für Helge sprechen, aber meiner Einschätzung ist, dass du mit dem Icon machen darfst, was du willst, solange du nicht behauptest es selbst erstellt zu haben oder versäumst zu erwähnen (in einer beiliegenden oder herunterladbaren Anleitung beispielsweise), dass das Icon Twitter gehört.

**Reply by Sorbat** [reply]
Hallo Bernhard, 

viele dank für deine Antwort. Das hört sich ja grundsätzlich erstmal nicht schlecht an. 

Habt Ihr im oB Projekt auch einen entsprechendes Verweis darauf? Ich hatte dazu nichts gefunden. Ich würde mich gerne daran orientieren.

**Reply by schlimmchen** [reply]
Ich kenne keinen ausdrücklichen Verweis. In den Quellen der Web-App siehst du aber, dass es sich um genau dieses Icon handelt.

https://github.com/helgeerbe/OpenDTU-OnBattery/blob/165a9bc16875acd4ee2adb8b77801a82a36a7975/webapp/src/components/NavBar.vue#L14-L16

Zusammen mit dem Wissen, dass es sich hier um Bootstrap und Vue handelt, kommt man dann zu den Links, die ich schon geteilt habe. Die sollten dann klar machen, was du mit diesem Icon tun kannst.
